### ISSUE TITLE
gsutil requires CRC32c mod

### ISSUE THREAD

**Issue by mtmorgan** [anvilproject/anvil-docker#25]
Reported by a participant during a PopUp workshop
```
> avworkspace(thousand_genomes)
[1] “anvil-datastorage/1000G-high-coverage-2019”
> dir.create(“1000G”)
> avtable(“sample”) %>% pull(gVCF) %>% head(1) %>% gsutil_cp(destination = “~/1000G”)
```
results in
```
CommandException: Downloading this composite object requires integrity checking with CRC32c,
but your crcmod installation isn’t using t...
```
which I think is a gsutil configuration step that should be performed on the image. See `gsutil help crcmod`

**Comment by rtitle** [comment]
May need to do something similar to this in the RStudio image: https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-base/Dockerfile#L140

**Comment by briandoconnor** [comment]
@rtitle , do you know if this is being worked on across the various images (RStudio, Jupyter, etc?). It seems like every image launched by Terra for interactive analysis should just have this installed. I believe it's installed in the Cromwell images used for batch analysis.

**Comment by rtitle** [comment]
Yes, we have this ticket to replicated recent fixes (including crcmod) to the RStudio image: https://broadworkbench.atlassian.net/browse/IA-2771
